src/Tools/Console/Command/DumpSchemaCommand.php

```diff
@@ -80,7 +80,7 @@ protected function configure(): void
     }
 
     /** @throws SchemaDumpRequiresNoMigrations */
-    public function execute(
```

Comment:
not sure hwy this method was public

not sure this was made on purpose, but this change might be a BC break in this cas